src/Submission/MapDataModel.jsx

```diff
@@ -151,6 +151,9 @@ class MapDataModel extends React.Component {
         if (data && data[this.state.parentNodeType]) {
           this.setState((prevState) => ({ validParentIds: data[prevState.parentNodeType] }));
         }
+      }).catch(error => {
+        // TODO - should we throw this error?
```

Comment:
@mfshao please let me know what you think of this one. Should we throw the error instead?

Reply:
printing is okay for me now